chainer/variable.py

```diff
@@ -240,9 +241,18 @@ def _set_grad_with_check(self, g, func, var):
         self._grad = g
 
 
+def _create_variable(
+        data, name, grad, initializer, update_rule, requires_grad):
+    return Variable(
+        data, name=name, grad=grad, initializer=initializer,
+        update_rule=update_rule, requires_grad=requires_grad)
+
+
 class Variable(object):
 
-    """Array with a structure to keep track of computation.
+    """__init__(data=None, name=None, grad=None, initializer=None, update_rule=None, requires_grad=True)
```

Comment:
Could you add `*,` between `data=None,` and `name=None`, like `data=None, *, name=None, ...`? It is a syntax of Py3 that indicates the following arguments are keyword only, and would be informative for users to notice that arguments after `data` cannot be passed as positional ones.